src/components/CreateRecipe.js

```diff
@@ -0,0 +1,343 @@
+import React, { Component } from "react";
+import { FiPlusCircle } from "react-icons/fi";
+import Grid from "@material-ui/core/Grid";
+import Typography from "@material-ui/core/Typography";
+import TextField from "@material-ui/core/TextField";
+import FormControlLabel from "@material-ui/core/FormControlLabel";
+import Checkbox from "@material-ui/core/Checkbox";
+import Button from "@material-ui/core/Button";
+import Slider from "@material-ui/core/Slider";
+import InputAdornment from "@material-ui/core/InputAdornment";
+import InputLabel from "@material-ui/core/InputLabel";
+import Select from "@material-ui/core/Select";
+import Paper from "@material-ui/core/Paper";
+import FormLabel from "@material-ui/core/FormLabel";
+import { GlobalCss } from "../materialUI/styles";
+
+export default class CreateRecipe extends Component {
+  handleNewRecipeSubmit = e => {
+    this.props.handleNewRecipeSubmit(e);
+  };
+
+  handleChange = e => {
+    this.props.handleChange(e);
+  };
+
+  addStep = e => {
+    this.props.addStep(e);
+  };
+
+  addIngredient = e => {
+    this.props.addIngredient(e);
+  };
+
+  handleCheckbox = e => {
+    this.props.handleCheckbox(e);
+  };
+
+  handleSlider(event, value) {
+    this.props.handleSlider(event, value);
+  }
+
+  render() {
+    const marks = [
+      {
+        value: 1,
+        label: "easy"
+      },
+      {
+        value: 2,
+        label: "advanced"
+      },
+      {
+        value: 3,
+        label: "difficult"
```

Comment:
Wollen wir die `difficulty` in Kleinbuchstaben machen? 🤔

Reply:
Ich änder es in Großbuchstaben 👍